dll/cpl/sysdm/environment.c

```diff
@@ -38,6 +38,7 @@ typedef struct _ENVIRONMENT_EDIT_DIALOG_DATA
     DWORD cyOld;
     DWORD dwDlgID;
     HWND hEditBox;
+    LPCWSTR Caption;
```

Comment:
Hungarian notation missing:
```suggestion
    LPCWSTR lpcwszCaption;
```

> Don't forget to refactor the code.